modules/cryptocoin.py

```diff
@@ -159,3 +159,36 @@ def coinPrice(irc, coin, amount, tick=True, bitfee=False):
                    prettify(float(info['percent_change_24h'])),
                    prettify(float(info['percent_change_7d'])))
     irc.reply(message + '.') 
+
+
+@commandHook(['ticker'])
+def ticker(irc, ev):
+    try:
+        coin = ev.args[0]
+    except (IndexError, ValueError):
+        coin = 'BTC'
+    try:
+        amount = float(ev.args[1])
+    except (IndexError, ValueError):
+        amount = 1.0
+    if coin.upper() == "DOGE":
```

Comment:
What if I want to know how much 1 doge is worth?

Put this if inside the except block. `if coin.upper() == 'DOGE': ... else amount=1.0`